llarp/service/endpoint.cpp

```diff
@@ -2114,6 +2191,20 @@ namespace llarp
       });
     }
 
+    void
+    Endpoint::UnmapRangeByExit(IPRange range, std::string exit)
+    {
+      // unmap all ranges that match the given exit when hot swapping
+      m_ExitMap.RemoveIf([&](const auto& item) -> bool {
+        if ((range.Contains(item.first)) and (item.second.ToString() == exit))
+        {
+          log::info(logcat, "{} unmap {} range mapping to exit node {}", Name(), item.first, exit);
+          return true;
+        }
+        return false;
+      });
```

Comment:
we should put the route poker down after doing the `RemoveIf` call if we no longer have anything in our exit map.